theme-contract/src/main/java/com/orange/ouds/theme/tokens/semantic/OudsSizeSemanticTokens.kt

```diff
@@ -1,101 +1,133 @@
+  val iconWithHeadingMediumSizeTallDesktop: Float = DimensionRawTokens.dimension650,
+  val iconWithHeadingMediumSizeTallMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingMediumSizeTallTablet: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingSmallSizeMediumDesktop: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingSmallSizeMediumMobile: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingSmallSizeMediumTablet: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingSmallSizeShortDesktop: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingSmallSizeShortMobile: Float = DimensionRawTokens.dimension400,
+  val iconWithHeadingSmallSizeShortTablet: Float = DimensionRawTokens.dimension400,
+  val iconWithHeadingSmallSizeTallDesktop: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingSmallSizeTallMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingSmallSizeTallTablet: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingXlargeSizeMediumDesktop: Float = DimensionRawTokens.dimension750,
+  val iconWithHeadingXlargeSizeMediumMobile: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingXlargeSizeMediumTablet: Float = DimensionRawTokens.dimension700,
+  val iconWithHeadingXlargeSizeShortDesktop: Float = DimensionRawTokens.dimension700,
+  val iconWithHeadingXlargeSizeShortMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingXlargeSizeShortTablet: Float = DimensionRawTokens.dimension650,
+  val iconWithHeadingXlargeSizeTallDesktop: Float = DimensionRawTokens.dimension800,
+  val iconWithHeadingXlargeSizeTallMobile: Float = DimensionRawTokens.dimension650,
+  val iconWithHeadingXlargeSizeTallTablet: Float = DimensionRawTokens.dimension800,
+  val iconWithLabelLargeSizeMedium: Float = DimensionRawTokens.dimension2xs,
+  val iconWithLabelLargeSizeShort: Float = DimensionRawTokens.dimension3xs,
+  val iconWithLabelLargeSizeShorter: Float = DimensionRawTokens.dimension4xs,
+  val iconWithLabelLargeSizeTall: Float = DimensionRawTokens.dimensionXs,
+  val iconWithLabelLargeSizeTaller: Float = DimensionRawTokens.dimensionLg,
+  val iconWithLabelMediumSizeMedium: Float = DimensionRawTokens.dimension4xs,
+  val iconWithLabelMediumSizeShort: Float = DimensionRawTokens.dimension5xs,
+  val iconWithLabelMediumSizeTall: Float = DimensionRawTokens.dimension3xs,
+  val iconWithLabelSmallSizeMedium: Float = DimensionRawTokens.dimension5xs,
+  val iconWithLabelSmallSizeShort: Float = DimensionRawTokens.dimension6xs,
+  val iconWithLabelSmallSizeTall: Float = DimensionRawTokens.dimension4xs,
+  val iconWithLabelXlargeSizeMedium: Float = DimensionRawTokens.dimensionMd,
+  val iconWithLabelXlargeSizeShort: Float = DimensionRawTokens.dimensionXs,
+  val iconWithLabelXlargeSizeTall: Float = DimensionRawTokens.dimensionLg,
+  val maxWidthGridGrid: Float = DimensionRawTokens.gridMaxWidth650,
```

Comment:
Spotted this one that shouldn't be in the list as it's web-specific. Closing the PR to fix in the parser.